src/harness/reference_models/ppa_model/utils.py

```diff
@@ -0,0 +1,154 @@
+#    Copyright 2016 SAS Project Authors. All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import numpy as np
+from area import area
+import geojson
+from shapely.geometry import shape
+from shapely.ops import cascaded_union
+from src.harness.util import getRandomLatLongInPolygon
+from collections import defaultdict
+from datetime import datetime
+import random
+import json
+import os
+import glob
+
+
+def to_dist_bear_vincenty(**kwargs):
+  """Placeholder to Call Vincenty, will be removed when the Vincenty is moved to reference models"""
+  pass
+
+
+def great_circle(**kwargs):
+  latitude = kwargs.pop('latitude')
+  longitude = kwargs.pop('longitude')
+  distance = kwargs.pop('distance') # In Km.
+  azimuth = kwargs.pop('azimuth')
+  calc = np.vectorize(to_dist_bear_vincenty)
```

Comment:
(Note Andy suggested to WG1 that we just not use vincenty and use haversine methods instead. I think that's the real solution to this vincenty craziness.)